### PR TITLE
Stats: Replace comparing text with icon and tooltips for the 7-day highlights section

### PR DESCRIPTION
**Pull request by dognose24**
#### Proposed Changes

* Introduce the `info` icon to replace the comparing text for the `7-day highlights`.
* Add tooltips with the comparing text and charts for the `info` icon.

#### Testing Instructions

* Spin up this change with the Calypso Live link.
* Navigate to the `Stats` > `Traffic` page.
* Ensure the comparing text on the heading of `7-day highlights` is replaced with the `info` icon.
* Ensure the tooltips for the `info` icon display in line with the design.

|Before|After|
|-|-|
|截圖 2023-02-06 上午10 49 49|截圖 2023-02-06 上午10 44 39|

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #70089

### CODE REVIEW

**Review by annacmc — APPROVED** [review]
Tested out on different browsers & screen sizes & looks good to me! 👍  

Not blocking, but I did notice the icon folds under the heading on smaller displays before I might expect it to:



Just double checking this is as intended.

**Comment by wilmersondasilva** [comment]
It worked as expected, and the code looks fine. I found the same behavior Anna described above, though.

**Review by wilmersondasilva — APPROVED** [review]

**Review by jsnmoon — CHANGES_REQUESTED** [review]
I see the same thing as Anna and Wilmerson at viewport <= 600px. This bug is caused by [these styles](https://github.com/Automattic/wp-calypso/blob/2b697c70d0d08eb64bd590ab0ee6e81c55b3b234/packages/components/src/highlight-cards/style.scss#L209) which we can safely remove (I think).

File: packages/components/src/highlight-cards/index.tsx [hunk 1]
```diff
@@ -1,10 +1,82 @@
...
+		<path
+			d="M120 6C120 4.89543 120.895 4 122 4H124C125.105 4 126 4.89543 126 6V34H120V6Z"
+			fill="#F6F7F7"
+		/>
+		<path
+			d="M130 11C130 9.89543 130.895 9 132 9H134C135.105 9 136 9.89543 136 11V34H130V11Z"
+			fill="#F6F7F7"
+		/>
+	</svg>
+);
```
Comment (jsnmoon) [inline packages/components/src/highlight-cards/index.tsx]: Could we move these SVG components into a separate file?

**Comment by dognose24** [comment]
> I see the same thing as Anna and Wilmerson at viewport <= 600px. This bug is caused by [these styles](https://github.com/Automattic/wp-calypso/blob/2b697c70d0d08eb64bd590ab0ee6e81c55b3b234/packages/components/src/highlight-cards/style.scss#L209) which we can safely remove (I think).

I removed the folding breakpoint in [48c4faf](https://github.com/Automattic/wp-calypso/pull/72987/commits/48c4fafa18722f724e703fbe17a5f251eff6f8a6), excluding the `small` element on mobile (width <= 480px) for [AnnualHighlightCards](https://github.com/Automattic/wp-calypso/blob/update/stats_comparing_info/packages/components/src/highlight-cards/annual-highlight-cards.tsx#L36-L40).

##### 7-day highlights on mobile
截圖 2023-02-08 下午12 11 33

##### Annual highlights on mobile
截圖 2023-02-08 下午12 11 23

File: packages/components/src/highlight-cards/index.tsx (hunk 1 above)
Comment (dognose24) [reply packages/components/src/highlight-cards/index.tsx]: Refactored in [80b67f7](https://github.com/Automattic/wp-calypso/pull/72987/commits/80b67f742ed2aac7abad2c2eb74ce299debb736c).

**Review by jsnmoon — APPROVED** [review]
Looks good 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7800694

Thank you @dognose24 for including a screenshot in the description! This is really helpful for our translators.

**Comment by jeffgolenski** [comment]
Looking good out in the world. Thanks, y'all!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7800694).

**Pull request merged**